README.md

```diff
@@ -55,6 +55,43 @@ Below is the basic idea how the website will work. We will also be working on ad
 
 We will also be working on adding more features to the Project.
 
+## Setup Guidelines
+Clone the repository -
+```
+git clone https://github.com/purnima143/Kurakoo.git
+```
+
+### Backend
+
+1. Run `cd server` on your CLI.
+
+2. Create a `.env` file and the format should be as given in `.env.example` in the server directory itself.
+
+3. Install the dependencies by running
+    ```
+    npm install
+    ```
+
+4. Run the server
+    ```
+    npm run dev
+    ```
+
+### Frontend
```

Comment:
@Mayank0255 Cloud You, please Remove unnecessary things from `client/README.md` and add this step as well.

Reply:
Sure